lib/logstash/inputs/beats.rb

```diff
@@ -37,6 +37,12 @@
 # a setting for the <<plugins-inputs-beats-type,`type`>> config option in
 # Logstash, it is ignored.
 #
+# IMPORTANT: If you are shipping events that span multiple lines, you need to
+# use the configuration options available in Beats to handle multiline events
```

Comment:
If I understand correctly, we can explicitly say "Filebeat" here since it's the only official Beat with multiline support.